tox.ini

```diff
@@ -60,10 +71,44 @@ deps =
 commands=
   pytest --basetemp={envtmpdir} {posargs}
 
+[testenv:win32-unittests]
```

Comment:
py35 makes us need to use 3.5 when we are using 3.6.

Reply:
We should be using the same version of Python no matter the platform in these environments. I think these environments actually need to get their Python version bumped to 3.6, since that's the actual version we install on the cluster, but that's unrelated to this change: https://github.com/dcos/dcos/blob/62901d36ca748e5a322170739e7c8c5a352e247f/packages/python/buildinfo.json#L10

My concern here is that we're consistent about the Python version we use to test, so I'd be fine with either using the existing 3.5 environments or, if your tests fail against that version, bumping everything to 3.6. I'd guess using the 3.5 environments would be the path of least resistance, I'm not sure if you've added anything that was introduced with 3.6.

Reply:
how about when you move Linux to 3.6 you merge these sections.
our CI is already configured for 3.6. This would be a step backwards.
everything else will be the same in the tox.ini file.